<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Opciones de Histogramas</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>87</width>
        <height>77</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="op1">
         <property name="text">
          <string>Equalizar</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="op2">
         <property name="text">
          <string>Desplazar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="op3">
         <property name="text">
          <string>Rangos</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajustes de Color</string>
     </property>
     <widget class="QSlider" name="rInf">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="rSup">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="gInf">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="gSup">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="bInf">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="bSup">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Rojo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Verde</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Azul</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Inf</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sup</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sup</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Inf</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sup</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Inf</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajustes Desplazar</string>
     </property>
     <widget class="QSlider" name="desp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="imgBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="original">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string>                                               original</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="result">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string>                                           Resultado</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>541</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="histogramO">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>4</y>
       <width>531</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string>                                                        Histograma Original</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>531</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="histogramR">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>4</y>
       <width>521</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string>                                                   Histograma Resultante</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir Ctrl+O</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
